{"@t":"2023-05-08T22:32:09.4952905Z","@mt":"HTTP {RequestMethod} {RequestPath} responded {StatusCode} in {Elapsed:0.0000} ms","@r":["9889.8741"],"@l":"Error","@x":"MixedDreams.Application.Exceptions.ModelValidationException: One or more validation failures have occurred.\r\n   at MixedDreams.WebAPI.Filters.ModelValidationFilterAttribute.OnActionExecuting(ActionExecutingContext context) in D:\\projects\\dotNet\\MixedDreams\\src\\MixedDreams.WebAPI\\Filters\\ModelValidationFilterAttribute.cs:line 20\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeNextActionFilterAsync()\r\n--- End of stack trace from previous location ---\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeInnerFilterAsync()\r\n--- End of stack trace from previous location ---\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeFilterPipelineAsync>g__Awaited|20_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)\r\n   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)\r\n   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)\r\n   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)\r\n   at Serilog.AspNetCore.RequestLoggingMiddleware.Invoke(HttpContext httpContext)","RequestMethod":"POST","RequestPath":"/api/auth/register/company","StatusCode":500,"Elapsed":9889.8741,"SourceContext":"Serilog.AspNetCore.RequestLoggingMiddleware","RequestId":"0HMQG2G4IGACR:00000001","ConnectionId":"0HMQG2G4IGACR"}
{"@t":"2023-05-08T22:32:11.7846762Z","@mt":"HTTP {RequestMethod} {RequestPath} responded {StatusCode} in {Elapsed:0.0000} ms","@r":["2294.1119"],"@l":"Error","@x":"MixedDreams.Application.Exceptions.ModelValidationException: One or more validation failures have occurred.\r\n   at MixedDreams.WebAPI.Filters.ModelValidationFilterAttribute.OnActionExecuting(ActionExecutingContext context) in D:\\projects\\dotNet\\MixedDreams\\src\\MixedDreams.WebAPI\\Filters\\ModelValidationFilterAttribute.cs:line 20\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeNextActionFilterAsync()\r\n--- End of stack trace from previous location ---\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeInnerFilterAsync()\r\n--- End of stack trace from previous location ---\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeFilterPipelineAsync>g__Awaited|20_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)\r\n   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)\r\n   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)\r\n   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)\r\n   at Serilog.AspNetCore.RequestLoggingMiddleware.Invoke(HttpContext httpContext)","RequestMethod":"POST","RequestPath":"/api/auth/register/company","StatusCode":500,"Elapsed":2294.1119,"SourceContext":"Serilog.AspNetCore.RequestLoggingMiddleware","RequestId":"0HMQG2G4IGACT:00000001","ConnectionId":"0HMQG2G4IGACT"}
{"@t":"2023-05-08T22:32:11.8681172Z","@mt":"Something went wrong: {@exception}","@l":"Warning","exception":"One or more validation failures have occurred.","SourceContext":"MixedDreams.WebAPI.Middlewares.ExceptionMiddleware","RequestId":"0HMQG2G4IGACT:00000001","RequestPath":"/api/auth/register/company","ConnectionId":"0HMQG2G4IGACT"}
{"@t":"2023-05-08T22:32:11.8681364Z","@mt":"Something went wrong: {@exception}","@l":"Warning","exception":"One or more validation failures have occurred.","SourceContext":"MixedDreams.WebAPI.Middlewares.ExceptionMiddleware","RequestId":"0HMQG2G4IGACR:00000001","RequestPath":"/api/auth/register/company","ConnectionId":"0HMQG2G4IGACR"}
{"@t":"2023-05-08T22:32:23.0805548Z","@mt":"HTTP {RequestMethod} {RequestPath} responded {StatusCode} in {Elapsed:0.0000} ms","@r":["6743.3492"],"@l":"Error","@x":"MixedDreams.Application.Exceptions.ModelValidationException: One or more validation failures have occurred.\r\n   at MixedDreams.WebAPI.Filters.ModelValidationFilterAttribute.OnActionExecuting(ActionExecutingContext context) in D:\\projects\\dotNet\\MixedDreams\\src\\MixedDreams.WebAPI\\Filters\\ModelValidationFilterAttribute.cs:line 20\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeNextActionFilterAsync()\r\n--- End of stack trace from previous location ---\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeInnerFilterAsync()\r\n--- End of stack trace from previous location ---\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeFilterPipelineAsync>g__Awaited|20_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)\r\n   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)\r\n   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)\r\n   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)\r\n   at Serilog.AspNetCore.RequestLoggingMiddleware.Invoke(HttpContext httpContext)","RequestMethod":"POST","RequestPath":"/api/auth/register/company","StatusCode":500,"Elapsed":6743.3492,"SourceContext":"Serilog.AspNetCore.RequestLoggingMiddleware","RequestId":"0HMQG2G4IGACT:00000002","ConnectionId":"0HMQG2G4IGACT"}
{"@t":"2023-05-08T22:32:23.1009993Z","@mt":"Something went wrong: {@exception}","@l":"Warning","exception":"One or more validation failures have occurred.","SourceContext":"MixedDreams.WebAPI.Middlewares.ExceptionMiddleware","RequestId":"0HMQG2G4IGACT:00000002","RequestPath":"/api/auth/register/company","ConnectionId":"0HMQG2G4IGACT"}
{"@t":"2023-05-08T22:34:57.8953712Z","@mt":"HTTP {RequestMethod} {RequestPath} responded {StatusCode} in {Elapsed:0.0000} ms","@r":["123128.6451"],"@l":"Error","@x":"MixedDreams.Application.Exceptions.ModelValidationException: One or more validation failures have occurred.\r\n   at MixedDreams.WebAPI.Filters.ModelValidationFilterAttribute.OnActionExecuting(ActionExecutingContext context) in D:\\projects\\dotNet\\MixedDreams\\src\\MixedDreams.WebAPI\\Filters\\ModelValidationFilterAttribute.cs:line 20\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeNextActionFilterAsync()\r\n--- End of stack trace from previous location ---\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeInnerFilterAsync()\r\n--- End of stack trace from previous location ---\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeFilterPipelineAsync>g__Awaited|20_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)\r\n   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)\r\n   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)\r\n   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)\r\n   at Serilog.AspNetCore.RequestLoggingMiddleware.Invoke(HttpContext httpContext)","RequestMethod":"POST","RequestPath":"/api/auth/register/company","StatusCode":500,"Elapsed":123128.6451,"SourceContext":"Serilog.AspNetCore.RequestLoggingMiddleware","RequestId":"0HMQG2G4IGACT:00000003","ConnectionId":"0HMQG2G4IGACT"}
{"@t":"2023-05-08T22:34:57.9159172Z","@mt":"Something went wrong: {@exception}","@l":"Warning","exception":"One or more validation failures have occurred.","SourceContext":"MixedDreams.WebAPI.Middlewares.ExceptionMiddleware","RequestId":"0HMQG2G4IGACT:00000003","RequestPath":"/api/auth/register/company","ConnectionId":"0HMQG2G4IGACT"}
{"@t":"2023-05-08T22:35:28.2246941Z","@mt":"HTTP {RequestMethod} {RequestPath} responded {StatusCode} in {Elapsed:0.0000} ms","@r":["27407.8591"],"@l":"Error","@x":"MixedDreams.Application.Exceptions.ModelValidationException: One or more validation failures have occurred.\r\n   at MixedDreams.WebAPI.Filters.ModelValidationFilterAttribute.OnActionExecuting(ActionExecutingContext context) in D:\\projects\\dotNet\\MixedDreams\\src\\MixedDreams.WebAPI\\Filters\\ModelValidationFilterAttribute.cs:line 20\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeNextActionFilterAsync()\r\n--- End of stack trace from previous location ---\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeInnerFilterAsync()\r\n--- End of stack trace from previous location ---\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeFilterPipelineAsync>g__Awaited|20_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)\r\n   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)\r\n   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)\r\n   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)\r\n   at Serilog.AspNetCore.RequestLoggingMiddleware.Invoke(HttpContext httpContext)","RequestMethod":"POST","RequestPath":"/api/auth/register/company","StatusCode":500,"Elapsed":27407.8591,"SourceContext":"Serilog.AspNetCore.RequestLoggingMiddleware","RequestId":"0HMQG2G4IGACT:00000004","ConnectionId":"0HMQG2G4IGACT"}
{"@t":"2023-05-08T22:35:28.2445820Z","@mt":"Something went wrong: {@exception}","@l":"Warning","exception":"One or more validation failures have occurred.","SourceContext":"MixedDreams.WebAPI.Middlewares.ExceptionMiddleware","RequestId":"0HMQG2G4IGACT:00000004","RequestPath":"/api/auth/register/company","ConnectionId":"0HMQG2G4IGACT"}
{"@t":"2023-05-08T22:35:41.3509685Z","@mt":"HTTP {RequestMethod} {RequestPath} responded {StatusCode} in {Elapsed:0.0000} ms","@r":["9280.8079"],"@l":"Error","@x":"MixedDreams.Application.Exceptions.ModelValidationException: One or more validation failures have occurred.\r\n   at MixedDreams.WebAPI.Filters.ModelValidationFilterAttribute.OnActionExecuting(ActionExecutingContext context) in D:\\projects\\dotNet\\MixedDreams\\src\\MixedDreams.WebAPI\\Filters\\ModelValidationFilterAttribute.cs:line 20\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeNextActionFilterAsync()\r\n--- End of stack trace from previous location ---\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeInnerFilterAsync()\r\n--- End of stack trace from previous location ---\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeFilterPipelineAsync>g__Awaited|20_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)\r\n   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)\r\n   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)\r\n   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)\r\n   at Serilog.AspNetCore.RequestLoggingMiddleware.Invoke(HttpContext httpContext)","RequestMethod":"POST","RequestPath":"/api/auth/register/company","StatusCode":500,"Elapsed":9280.8079,"SourceContext":"Serilog.AspNetCore.RequestLoggingMiddleware","RequestId":"0HMQG2G4IGACT:00000005","ConnectionId":"0HMQG2G4IGACT"}
{"@t":"2023-05-08T22:35:41.3721806Z","@mt":"Something went wrong: {@exception}","@l":"Warning","exception":"One or more validation failures have occurred.","SourceContext":"MixedDreams.WebAPI.Middlewares.ExceptionMiddleware","RequestId":"0HMQG2G4IGACT:00000005","RequestPath":"/api/auth/register/company","ConnectionId":"0HMQG2G4IGACT"}
{"@t":"2023-05-08T22:36:17.8741902Z","@mt":"HTTP {RequestMethod} {RequestPath} responded {StatusCode} in {Elapsed:0.0000} ms","@r":["34790.1220"],"@l":"Error","@x":"MixedDreams.Application.Exceptions.ModelValidationException: One or more validation failures have occurred.\r\n   at MixedDreams.WebAPI.Filters.ModelValidationFilterAttribute.OnActionExecuting(ActionExecutingContext context) in D:\\projects\\dotNet\\MixedDreams\\src\\MixedDreams.WebAPI\\Filters\\ModelValidationFilterAttribute.cs:line 20\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeNextActionFilterAsync()\r\n--- End of stack trace from previous location ---\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeInnerFilterAsync()\r\n--- End of stack trace from previous location ---\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeFilterPipelineAsync>g__Awaited|20_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)\r\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)\r\n   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)\r\n   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)\r\n   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)\r\n   at Serilog.AspNetCore.RequestLoggingMiddleware.Invoke(HttpContext httpContext)","RequestMethod":"POST","RequestPath":"/api/auth/register/company","StatusCode":500,"Elapsed":34790.122,"SourceContext":"Serilog.AspNetCore.RequestLoggingMiddleware","RequestId":"0HMQG2G4IGACT:00000006","ConnectionId":"0HMQG2G4IGACT"}
{"@t":"2023-05-08T22:36:17.8934039Z","@mt":"Something went wrong: {@exception}","@l":"Warning","exception":"One or more validation failures have occurred.","SourceContext":"MixedDreams.WebAPI.Middlewares.ExceptionMiddleware","RequestId":"0HMQG2G4IGACT:00000006","RequestPath":"/api/auth/register/company","ConnectionId":"0HMQG2G4IGACT"}
